node/pkg/driver/device_connectivity/errors.go

```diff
@@ -13,6 +13,15 @@ func (e *MultipleDmDevicesError) Error() string {
 	return fmt.Sprintf("Detected more than one multipath device (%v) for single volume (%s)", e.MultipathDevices, e.VolumeId)
 }
 
+type MultipleVolumeIdsError struct {
+	VolumeIds           map[string]bool
+	MultipathDeviceName string
+}
+
+func (e *MultipleVolumeIdsError) Error() string {
+	return fmt.Sprintf("Detected more than one volume uuids (%v) for single multipath device name (%s)", e.VolumeIds, e.MultipathDeviceName)
```

Comment:
```suggestion
	return fmt.Sprintf("Detected more than one volume uuid (%v) for a single multipath device name (%s)", e.VolumeIds, e.MultipathDeviceName)
```

Reply:
FYI you may resolve yourself suggestion conversations that are only about the single suggested change (like this one).
i.e. if the suggestion is specific only to the previewed diff (not a general suggestion), then the reviewee may resolve it as well, since the chance of a miss is negligible